### PR TITLE
Add section for DB Loader

### PR DESCRIPTION
**Pull request by luisquintanilla**
Add section for DB Loader

[Preview Link](https://review.docs.microsoft.com/en-us/dotnet/machine-learning/how-to-guides/load-data-ml-net?branch=pr-en-us-14272)

[AB#1594300](https://dev.azure.com/mseng/677da0fb-b067-4f77-b89b-f32c12bb8617/_workitems/edit/1594300)

### CODE REVIEW

**Comment by luisquintanilla** [comment]
Ready for review when you get a chance @natke @CESARDELATORRE

File: docs/machine-learning/how-to-guides/load-data-ml-net.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Load data from files and other sources
 description: This how-to shows you how to load data for processing and training into ML.NET. The data is originally stored in files or other data sources such as databases, JSON, XML or in-memory collections.
-ms.date: 08/01/2019
```
Comment (natke) [inline docs/machine-learning/how-to-guides/load-data-ml-net.md]: Did we forget author here?

**Review by natke — APPROVED** [review]
LGTM!

File: docs/machine-learning/how-to-guides/load-data-ml-net.md (hunk 1 above)
Comment (luisquintanilla) [reply docs/machine-learning/how-to-guides/load-data-ml-net.md]: Couldn't think of a clever pen name. Fixed 🙂

**Pull request merged**